AjobK/Athena
# 206 follow system

closes #206 

**What is it supposed to do**
Users should be able to follow other users to see activity in the future

**How can we test your branch**
1. Go to a profile
2. Click the follow button
3. Check if the follow is still active when moving around in the app
4. Unfollow
5. Check if the unfollow is still active when moving around in the app
6. Check responsiveness
7. Etc.


**What broweser have been checked**
